packages/tasit-sdk/src/testHelpers/helpers.js

```diff
@@ -0,0 +1,123 @@
+import { Account, Action } from "../TasitSdk";
+const { Contract, NFT } = Action;
+
+import { abi as manaABI } from "./MANAToken.json";
+import { abi as landABI } from "./LANDRegistry.json";
+import { abi as landProxyABI } from "./LANDProxy.json";
+import { abi as estateABI } from "./EstateRegistry.json";
+import { abi as markplaceABI } from "./Marketplace.json";
+
+const manaAddress = "0xb32939da0c44bf255b9810421a55be095f9bb3f4";
+const landAddress = "0x6191bc768c2339da9eab9e589fc8bf0b3ab80975";
+const landProxyAddress = "0x773f11ed472aa43e4ebaa963bcfbbea5a10c1bbd";
+const estateAddress = "0x41b598a2c618b59b74540ac3afffb32f7971b37a";
+const marketplaceAddress = "0x07c0e972064e5c05f7b3596d81de1afd35457eae";
+const fullNFTAddress = "0x0E86f209729bf54763789CDBcA9E8b94f0FD5333";
+
+// TODO: Go deep on gas handling.
+// Without that, VM returns a revert error instead of out of gas error.
+// See: https://github.com/tasitlabs/TasitSDK/issues/173
+const gasParams = {
+  gasLimit: 7e6,
+  gasPrice: 1e9,
+};
+
+const setupContracts = async owner => {
+  // Note: It would be cooler to use NFT here if
+  // Decentraland Land contract followed ERC721 exactly
```

Comment:
This comment belongs before the NFT contracts - just land and estate. Not MANA, which is ERC20

Reply:
Oh, ok.

Reply:
Done

Reply:
Great. Resolving!